node_package/src/context.ts

```
@@ -1,9 +1,36 @@
+import { ReactOnRails as ReactOnRailsType } from './types';
+import * as ClientStartup from './clientStartup';
+
 /**
  * Get the context, be it window or global
  * @returns {boolean|Window|*|context}
  */
-export default function context(this: void): Window | NodeJS.Global | void {
+export function context(this: void): Window | NodeJS.Global | void {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)